installers/docker-compose/templates/.env

```diff
@@ -0,0 +1,42 @@
+BUILD_DIR=__BUILD_DIR__
+
+# Java
+TRUSTSTORE_PASSWORD=changeit
+
+# Mongo
+MONGO_SERVICE_NAME=mongod
```

Comment:
That's fair, though I would like to keep these around in case we want to write some doc on how to debug, like we can tell people to do `docker container exec -it sdlc sh`. I will think more about this.